test/fixtures/rides.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
```

Comment:
What exactly would fixtures do?

Reply:
This is default test content produced by rails when you generate a model so don't worry about this right now!